### PR TITLE
Add HW4 Bar with Executor Service

### PR DESCRIPTION
**Pull request by 1stnik**

### CODE REVIEW

File: HW4_Bar_Executor_Service/src/main/java/com/hillel/Main.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                executorService.submit(bartender);
+                try {
+                    Thread.sleep(200);
+                } catch (InterruptedException e) {
+                    throw new RuntimeException(e);
+                }
+            }
+            executorService.shutdown();
+        }
+        System.exit(0);
```
Comment (kartius) [inline HW4_Bar_Executor_Service/src/main/java/com/hillel/Main.java:45]: What for did you use System.exit(0) here?

File: HW4_Bar_Executor_Service/src/main/java/com/hillel/Main.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            for (int i = 1; i <= numOfBartenders; i++) {
+                Bartender bartender = new Bartender(bar, "Bartender" + i);
+                executorService.submit(bartender);
+                try {
+                    Thread.sleep(200);
+                } catch (InterruptedException e) {
+                    throw new RuntimeException(e);
+                }
+            }
+            executorService.shutdown();
```
Comment (kartius) [inline HW4_Bar_Executor_Service/src/main/java/com/hillel/Main.java:43]: use shutdown in finally block

File: HW4_Bar_Executor_Service/src/main/java/com/hillel/Main.java (hunk 1 above)
Comment (1stnik) [reply HW4_Bar_Executor_Service/src/main/java/com/hillel/Main.java:45]: The program continues working without it.

File: HW4_Bar_Executor_Service/src/main/java/com/hillel/Main.java (hunk 2 above)
Comment (1stnik) [reply HW4_Bar_Executor_Service/src/main/java/com/hillel/Main.java:43]: Unfortunately, my JDK doesn't allow me to use try-with-resources or try-catch-finally with ExecutorService class. I'm to solve that problem.

Comment (1stnik) [reply HW4_Bar_Executor_Service/src/main/java/com/hillel/Main.java:43]: Oh, you're right. I did it.

**Pull request merged**